xbmc/network/WebServer.cpp

```diff
@@ -102,9 +102,25 @@ CWebServer::CWebServer()
 #endif
 }
 
+static MHD_Response* create_response(size_t size, void* data, int free, int copy)
+{
+#if (MHD_VERSION >= 0x00094001)
+  MHD_ResponseMemoryMode mode = MHD_RESPMEM_PERSISTENT;
+  if (copy)
+    mode = MHD_RESPMEM_MUST_COPY;
+  else if (free)
+    mode = MHD_RESPMEM_MUST_FREE;
+  struct MHD_Response *response = MHD_create_response_from_buffer(0, nullptr, mode);
```

Comment:
I would just return here (and two lines down as well) because otherwise we run the danger of someone changing the name in the `#if` part and not in the `#else` part.